### ISSUE TITLE
Would like to have control over window location...

### ISSUE THREAD

**Issue by precisionpete** [wailsapp/wails#2988]
### Is your feature request related to a problem? Please describe.

It would be nice to have control over where the window opens. 

e.g. The first time it opens, it opens in the center of the screen. After hiding and showing it, it opens near the top left (under Linux at least). I would like to be able to pin it to the bottom right - like Microsoft OneDrive. I am using a port of lantern systray to create a panel icon to control the show/hide of the window. Works great. I would like to be able to have the window show near the icon.

e.g. model behavior after MS OneDrive on Windows.

### Describe the solution you'd like

Prefer control over properties that will be remembered between show/hide.

I don't need pixel control. Aligning center, top-left, bottom-left, etc would be enough. Maybe if the user moves it, it will remember the location. Or have the ability to lock it somewhere.

It would have to be different or computable based on some criteria. e.g. Windows = bottom left, Mac = top right, Linux needs somehow to figure that out from the environment. Cinnamon, vs PopOS, etc.

**Comment by precisionpete** [comment]
Interesting.

How would I backport? fork it and snake the changes on my copy?

*Peter Carroll*
***@***.***
https://bluefriday.ca

**Comment by PylotLight** [comment]
> Interesting. How would I backport? fork it and snake the changes on my copy?
Correct, submit a new PR with the changes in V2 in your fork, to have the changes considered and merged.
https://wails.io/community-guide#ways-of-contributing

**Comment by Mluocheng** [comment]
我的解决方案是 默认隐藏的，在前端入口出重新设置位置：        const x = (screen.width - 应用width) * window.devicePixelRatio - 16
        const y = (screen.height - 应用高度) * window.devicePixelRatio - 76
        window.runtime.WindowSetPosition(x, y)

**Comment by leaanthony** [comment]
Hey there\!

First off, thank you for taking the time to open this issue. Community feedback like yours has been instrumental in shaping Wails into what it is today.

We're doing some housekeeping on our issue tracker as we focus on the v3 release. This issue has been open for a while, and honestly, with the project's growth and the sheer volume of feedback we receive, some things inevitably slip through the cracks. That's on us, not you.

We're closing this issue, but that doesn't mean it wasn't valuable. If this is still relevant:

- For bugs: Please test with the latest version (or v3 alpha) and open a fresh issue if it persists
- For features: Feel free to open a new issue with updated context

Thank you for being part of the Wails journey. Your patience and support mean more than you know.
